src/Centreon/Domain/Menu/Model/Page.php

```diff
@@ -107,4 +109,23 @@ public function isReact(): bool
     {
         return $this->isReact;
     }
+
+    /**
+     * Return the redirection uri of the page.
+     *
+     * @return string
+     */
+    public function getRedirectionUri(): string
+    {
+        if ($this->isReact === true) {
```

Comment:
As mentionned last time, if I do remember well we can keep the boolean equality when it is false as for true it seems quite "obvious" as the variable is prefixed by "is"

```suggestion
        if ($this->isReact) {
```

Reply:
done